truffle.js

```diff
@@ -3,13 +3,16 @@ require("babel-register");
 require("babel-polyfill");
 const TestRPC = require("ethereumjs-testrpc");
 
-const providerUrl = "http://localhost:8545";
-const nanoPath = "44'/60'/0'/0`";
-
-const nanoProvider = () =>
-  process.argv.filter(arg => arg.includes("nano")).length > 0
-    ? require("./nanoWeb3Provider").nanoWeb3Provider(providerUrl, nanoPath)
-    : undefined;
+const nanoProvider = (providerUrl, nanoPath, network) => {
+  for (let i = 0; i < process.argv.length; i += 1) {
```

Comment:
why don't you use nice list operations like `some`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some then it is one liner

Reply:
Looks better indeed